src/components/ClientPage/ClientPage.module.css

```diff
@@ -0,0 +1,140 @@
+.clientPage {
+    background-color: #525f7f;
```

Comment:
It would be nice to use Variable for color.

Reply:
yeah, it would be nice xD kidding, thank you i will change it